### PR TITLE
Activation of sapling for ZClassic

### PR DESCRIPTION
**Pull request by jeromemorignot**
https://medium.com/@skellers/zclassic-sapling-the-new-core-d0eae720d826

sapling & overwinter will be active after block 476969

### CODE REVIEW

File: coins/zcl.json [hunk 1]
```diff
@@ -5,6 +5,8 @@
     "requireShielding": true,
     "peerMagic": "24e92764",
     "txfee": 0.0004,
+    "sapling": 476969,
+    "overwinter": 476969,
```
Comment (egyptianbman) [inline coins/zcl.json]:
```suggestion
```
This value isn't actually required. Can you remove it?

Comment (egyptianbman) [reply coins/zcl.json]: Actually, you can set `sapling` to just true since that height has passed.

Comment (jeromemorignot) [reply coins/zcl.json]: Yes right :) At the time it was not

**Review by egyptianbman — APPROVED** [review]
Thank you!

**Pull request merged**